### ISSUE TITLE
php_docblock_checker: Data too long for column 'meta_value'

### ISSUE THREAD

**Issue by tobiastom** [Block8/PHPCI#572]
I just enabled the php_docblock_checker plugin and now I get the following error:

```
RUNNING PLUGIN: php_docblock_checker
EXCEPTION: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'meta_value' at row 1
PLUGIN STATUS: FAILED
```

`meta_value` is a TEXT column, obviously without a length.

**Comment by SamMousa** [comment]
My pull request #745 solves this issue.

**Comment by tvbeek** [comment]
Closed with #540
